build-scripts/rcm-guest/publish-oc-binary.sh

```diff
@@ -12,7 +12,7 @@ extract() {
     mkdir macosx windows
     for arch in x86_64 ${ARCH}; do
         rpm=$(echo "$(rpm_name "${arch}")"*)
-        [ "${arch}" != x86_64 && ! -e "${rpm}" ] && continue
+        [ "${arch}" != x86_64 -a ! -e "${rpm}" ] && continue
```

Comment:
`[[ "${arch}" != x86_64 && ! -e "${rpm}" ]] && continue` ?

Reply:
@bbguimaraes or you could do `[ "${arch}" != x86_64 ] && [ ! -e "${rpm}" ] && continue` but I wouldn't :D

Reply:
@bbguimaraes Actually, the modeline is `#!/bin/sh`, so your fix is most proper, though I would quote both sides of the string comparison.